CollAction/Frontend/app/global/CookieMessage.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import renderComponentIf from "./renderComponentIf";
+
+interface ICookieMessageProps {
+    canTrack: boolean;
+    cookieString: string;
+}
+
+interface ICookieMessageState {
+    showBanner: boolean;
+}
+
+export default class CookieMessage extends React.Component<ICookieMessageProps, ICookieMessageState> {
+    constructor(props) {
+        super(props);
+        this.state = { showBanner: false };
+    }
+
+    componentDidMount() {
+        this.setState({
+            showBanner: !this.props.canTrack
+        });
+    }
+
+    accept() {
+        document.cookie = this.props.cookieString;
+        location.reload();
+    };
+
+    renderMessage() {
+        return (
+            <div>
+                <div id="cookie-message-banner">
+                    <div className="container">
+                        <div className="row">
+                            <div className="col-xs-12">
+                                <p>Thank you for visiting our website! Please note that our website uses cookies to analyze and improve the performance of our website and to make social media integration possible. For more information on the use of cookies and privacy related matters, please see our <a href="home/privacy">Privacy and Cookies Policy</a>. By clicking "I accept" or by using this site, you consent to the use of cookies.</p>
```

Comment:
I think this is still missing a few things. I thought this was a good summary: https://www.cookiebot.com/en/gdpr-cookies/

* User should be able to reject the non-essential cookies (for example: tracking, youtube)
* I think this should be a bit clearer, maybe a bullet point list with a link to the full privacy policy?

Reply:
@ronvdakker can you comment on this? See if we need different copy here.

Do we want to give users an option to either opt-in or opt-out of tracking cookies. Perhaps you can create a new story for that?

Reply:
Yes, we can create a separate story for the opt in and opt out of essentials. 

Lets go ahead with this one for now though, and improve upon it later.